src/saturn_engine/worker/executors/queue.py

```diff
@@ -45,6 +55,44 @@ def start(self) -> None:
         self.submit_tasks.start()
         self.processing_tasks.start()
 
+    async def _process_error(
+        self, processable: ExecutableMessage, exception: Exception
+    ):
+        # Match the exception and consume it if found, else reraise the exception
+        for k, ts in processable.output.items():
+            # TODO: better way to identify error output, maybe seperate field instead
+            if is_output_error_handler(k):
+                exc_filter = parse_warning_filter(k, escape=False)
+                if exc_filter.action == "ignore":
+                    raise HandledException("Exception ignored")
+
+                exc_details = get_exception_details()
+                if does_output_match(exc_details, exc_filter):
+                    self.consuming_tasks.create_task(
+                        self.consume_output(
+                            processable=processable,
+                            output=[
+                                PipelineOutput(
```

Comment:
You could just return this output and let the regular calling code consume output code take care of consuming it.

ie (in executore.run_queue)

```python
                    async def scope(
                        xmsg: ExecutableMessage,
                    ) -> Optional[PipelineResults]:
                        try:
                            return await self.executor.process_message(xmsg.message)
                        except Exception as err:
                            return await self._process_error(xmsg, err)
```